search-api/search/cardTemplates/models.py

```diff
@@ -0,0 +1,15 @@
+from search import db
+
+class Something(db.Model):    
```

Comment:
We need to remove these before merging (assuming Something and Otherthing are just for testing.)

Reply:
For reference purpose I've commented them.